microcosm_pubsub/daemon.py

```diff
@@ -82,7 +82,7 @@ def defaults(self):
         if self.sqs_event:
             config.update(
                 sqs_envelope=dict(
-                    strategy_name=NaiveSQSEnvelope.__name__,
+                    strategy_name=CodecSQSEnvelope.__name__,
```

Comment:
Is this backwards-compatible with a daemon deployed as currently? If not then this will make deployment to Lambda more complicated and risk breaking existing daemons if they upgrade the library version. Not necessarily a problem, but:

* if we can maintain backwards-compatibility then we should
* if we can't then we need a major version number bump and some more planning 😄

Reply:
`sqs_event` won't be set on any daemons currently deployed other than the daemons that @andreyshuster is testing with. The pre-cursor of this work is already present in the release that added sentry support. 

https://github.com/globality-corp/microcosm-pubsub/pull/238/files#diff-063fc687b977ac99757488ad2741ee96R82

Reply:
Thanks. With hindsight `sqs_event` could have been clearer (at least for this 🐻 of very little brain) 😄.